packages/components/src/FormField/FormField.css

```diff
@@ -60,6 +60,12 @@
   box-shadow: var(--shadow-focus);
 }
 
+/* override iOS default styling for disabled input */
+.wrapper input:disabled {
+  -webkit-text-fill-color: var(--field--value-color);
```

Comment:
nit: does it make sense to move this down to the `disabled` section around like 86? It's technically a `wrapper` selector but functionally it's performing the same job as the `disabled` styling

Reply:
I guess less of a nit and more of a discussion

Reply:
tru, would just have to write it as 

```css
.disabled input:disabled {}
```

Reply:
Just pulling down all the newness in my local to test, and once Cloudflare is finished I'll give it a whirl on my phone + iPad

Reply:
Or i guess 

```css
.disabled :disabled {}
```

Because `textarea` is a thing.  _And yes, with the space in between_